tests/foreman/cli/test_discoveredhost.py

```diff
@@ -73,6 +75,11 @@ def setUpClass(cls):
 
         # Build PXE default template to get default PXE file
         Template.build_pxe_default()
+        # let's just modify the timeouts to speed things up
+        ssh.command("sed -ie 's/TIMEOUT [[:digit:]]\+/TIMEOUT 1/g'"
+                    "/var/lib/tftpboot/pxelinux.cfg/default")
+        ssh.command("sed -ie '/APPEND initrd/s/$/ fdi.countdown=1/'"
+                    "/var/lib/tftpboot/pxelinux.cfg/default")
```

Comment:
by updating the timeouts in rendered pxelinux config directly on TFTPboot we don't need to unlock/clone and update the actual template